Core/include/Acts/Geometry/CylinderVolumeStack.hpp

```diff
@@ -0,0 +1,91 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Geometry/CylinderVolumeBounds.hpp"
+#include "Acts/Geometry/Volume.hpp"
+#include "Acts/Utilities/BinningType.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <vector>
+
+namespace Acts {
+
+class CylinderVolumeStack : public Volume {
```

Comment:
should we put some doc on this class?

Reply:
I thought I pushed this already... where did it go? 🧐

Reply:
Added the comments now.